docs/source/user-guide/tutorials/build-nltk.rst

```diff
@@ -0,0 +1,76 @@
+=================================
+Building a conda package for NLTK
+=================================
+
+The Natural Language Toolkit (NLTK) is a platform used for building Python 
+programs that apply statistical natural language processing (NLP) to human 
+language data. It can be difficult to install, but is easy when you build this 
+NLTK conda package.
+
+This package also includes all of the NLTK data sets, approximately 8 GB of
+data.
+
+Download the following files: https://gist.github.com/danielfrg/d17ffffe0dc8ed56712a0470169ff546
```

Comment:
we should host this differently, rather than some random gist on Daniel's personal account.